py/server/deephaven/server/plugin/object/__init__.py

```diff
@@ -27,7 +28,7 @@ class ExporterAdapter(Exporter):
     def __init__(self, exporter: _JExporterAdapter):
         self._exporter = exporter
 
-    def reference(self, object, allow_unknown_type : bool = False, force_new : bool = False) -> Optional[Reference]:
+    def reference(self, object, allow_unknown_type: bool = False, force_new: bool = False) -> Optional[Reference]:
```

Comment:
need to change defaults to True, so that by default at least plugins don't break